ui/src/components/setting/SettingWebhook.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+  <fragment>
+    <v-form>
+      <ValidationObserver
+        ref="data"
+        v-slot="{ passes }"
+      >
+        <v-row>
+          <v-col>
+            <h3>
+              Webhook
+            </h3>
+          </v-col>
+          <v-spacer />
+          <v-col
+            md="auto"
+            class="ml-auto"
+          >
+            <v-btn
+              v-if="!editWebhook"
+              outlined
+              @click="editWebhook = !editWebhook"
+            >
+              Change webhook
+            </v-btn>
+            <div
+              v-if="editWebhook"
+            >
+              <v-btn
+                class="mr-2"
+                outlined
+                @click="cancel"
+              >
+                Cancel
+              </v-btn>
+              <v-btn
+                outlined
+                @click="passes(save)"
+              >
+                Save
+              </v-btn>
+            </div>
+          </v-col>
+        </v-row>
+        <div
+          class="pl-4 pr-4"
```

Comment:
You can create another external div and put the class,  because the class is repeated in the line 69.